src/test/ui/associated-types/associated-types-eq-3.stderr

```diff
@@ -36,9 +36,7 @@ error[E0271]: type mismatch resolving `<isize as Foo>::A == Bar`
   --> $DIR/associated-types-eq-3.rs:40:9
    |
 LL |     baz(&a);
-   |     --- ^^ type mismatch resolving `<isize as Foo>::A == Bar`
-   |     |
-   |     required by a bound introduced by this call
```

Comment:
This went away because `<isize as Foo>::A == Bar` is not a requirement on `baz`, and the fact that we were reporting it as such was a bug.